openshift/kustomize/services/build/base/capture.yaml

```diff
@@ -6,13 +6,13 @@ metadata:
   name: capture-service
   annotations:
     description: Destination for built images.
+    created-by: jeremy.foster
   labels:
     name: capture-service
     part-of: tno
     version: 1.0.0
     component: capture
     managed-by: kustomize
-    created-by: jeremy.foster
```

Comment:
Moved to annotation so that it can support multiple entries.  Did this on all templates.